changelogs/unreleased/gh-5385-tiny-tuples-v3.md

```diff
@@ -1,3 +1,4 @@
 ## feature/core
 
-* Introduce compact tuples that allow to save 4 bytes per tuple in case of small userdata  (gh-5385)
+* Introduce compact tuples that allow saving 4 bytes per tuple
+  in case of small userdata (gh-5385).
```

Comment:
Nit: It is better to don't make text much shorter than 80 symbols to make the entire changelog look visually same.